src/Microsoft.AspNetCore.Razor.Evolution/RazorEngine.cs

```diff
@@ -16,10 +16,24 @@ public static RazorEngine Create()
         public static RazorEngine Create(Action<IRazorEngineBuilder> configure)
         {
             var builder = new DefaultRazorEngineBuilder();
+            AddDefaults(builder);
             configure?.Invoke(builder);
             return builder.Build();
         }
 
+        public static RazorEngine CreateEmpty(Action<IRazorEngineBuilder> configure)
+        {
+            var builder = new DefaultRazorEngineBuilder();
+            configure?.Invoke(builder);
+            return builder.Build();
+        }
+
+        internal static void AddDefaults(IRazorEngineBuilder builder)
```

Comment:
This is a good idea, I'm banking it along with my other good ideas for later. Not relevant until we need it.